src/redux/configureStore.js

```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux';
+import booksReducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+const allReducers = combineReducers({
+  booksReducer,
+  categoriesReducer,
+});
+
+export default allReducers;
```

Comment:
- [ ] You didnt import configureStore from redux toolkit. Please update it. It is part of the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Importing%20the%20necessary%20methods%20from%20Redux%20Toolkit.)
- [ ] Please update the readme file, ı wrote details review on the summary.